Docker/build/Wikibase/default.env

```diff
@@ -0,0 +1,6 @@
+MW_SITE_NAME=wikibase-docker
```

Comment:
These are still hardcoded in the dockerfile :facepalm:

Reply:
Have you seen my recent ticket https://phabricator.wikimedia.org/T297469 about this? :-)